packages/app/pages/index.tsx

```diff
@@ -210,19 +241,23 @@ const MembershipCard = (props: IMembershipCardProps) => {
             />
             <ModalInputBox>
               <ModalInputLabel htmlFor="bcolor">Brand color:</ModalInputLabel>
-              <input type="color" 
-                id="bcolor" 
-                name="bcolor" 
+              <input
+                type="color"
+                id="bcolor"
+                name="bcolor"
                 value={brandColor || "#fdf3e7"}
-                onChange={(e) => setBrandColor(e.target.value)}></input>
+                onChange={(e) => setBrandColor(e.target.value)}
+              ></input>
             </ModalInputBox>
             <ModalInputBox>
               <ModalInputLabel htmlFor="acolor">Accent color:</ModalInputLabel>
-              <input type="color" 
-                id="acolor" 
-                name="acolor" 
+              <input
+                type="color"
+                id="acolor"
+                name="acolor"
                 value={accColor || "#324841"}
-                onChange={(e) => setAccColor(e.target.value)}></input>
+                onChange={(e) => setAccColor(e.target.value)}
+              ></input>
```

Comment:
sorry about this noise, I have an autoformatter in my code editor that formats on save